.travis.yml

```diff
@@ -14,3 +17,7 @@ deploy:
   gem: kontena-plugin-shell
   on:
     tags: true
+after_deploy:
```

Comment:
I think the guard is more of an annoyance if that after_deploy thing only fires after a gem deploy, which happens only for tagged commits.

In kontena repo we often make releases from some release branches or PR's, so in this repo it won't work on anything but master.